app/src/androidTest/java/com/github/se/signify/ui/screens/profile/MyStatsScreenTest.kt

```diff
@@ -40,46 +46,58 @@ class MyStatsScreenTest {
     // Verify top blue bar is displayed
     composeTestRule.onNodeWithTag("TopBlueBar").assertIsDisplayed()
 
-    // Verify days count is displayed with correct text
-    composeTestRule.onNodeWithTag("flameIcon").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("numberOfDays").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("numberOfDays").assertTextEquals("10 days")
+    // Verify top information are displayed correctly
```

Comment:
It looks like you've added checks here. Don't get too distracted :) , sometimes you just have to let it be.